README.md

```diff
@@ -6,62 +12,59 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/1342f842a14f40cca856d6e81204f8ac)](https://app.codacy.com/gh/Sudokuru/Frontend/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Maintainability](https://api.codeclimate.com/v1/badges/4cd1d1027b7b2532c680/maintainability)](https://codeclimate.com/github/Sudokuru/Frontend/maintainability)
 
-# Start the frontend app
-
-Run `npm i` in the root folder  
-Run `npm run start`
+## Sudokuru is an open-source project focused on developing a world-class, cross-platform Sudoku app. We aim to provide a delightful user experience while also contributing to the community by building a collection of reusable software modules. These modules are designed to be free, well-documented, modern, and interoperable, allowing developers to easily incorporate them into their own Sudoku-related projects.
 
-For testing with a mobile device you can download the Expo app.  
-You will also need to be logged in to Expo on the development device (laptop/desktop that is running the code).  
-To login type: `expo login` and fill out the parameters.
+### This frontend module is the primary user interface for Sudokuru, providing a cross-platform Sudoku experience built with React Native Web and TypeScript. It offers a clean, intuitive design with features such as:
 
-Contact Thomas to get added to the Expo organization so that you can log in with your own email and password.
+- 🎮 Play Sudoku across 9 different difficulty levels generated by the Sudokuru Clearinghouse module
+- 🎓 Learn how to play Sudoku with lessons from the basics all the way to advanced strategies
```

Comment:
Maybe rephrase this to:
Sudoku lessons from basic to advanced strategies

Reply:
or maybe "Learn how to play Sudoku with lessons for simple and advanced strategies"